src/web/client/common/portalSchema.ts

```diff
@@ -0,0 +1,6264 @@
+export const portal_schema_data = {
+    "entities":{
+       "organization":[
```

Comment:
Why is this an array?

Reply:
this is same as entities.fields , keepign the schema str intact